proposals/load_stac.json

```
@@ -186,7 +186,7 @@
         },
         {
             "name": "properties",
-            "description": "Limits the data by metadata properties to include only data in the data cube which all given conditions return `true` for (AND operation).\n\nSpecify key-value-pairs with the key being the name of the metadata property, which can be retrieved with the openEO Data Discovery for Collections. The value must be a condition (user-defined process) to be evaluated against a STAC API. This parameter is not supported for static STAC.",
+            "description": "Limits the data by metadata properties to include only data in the data cube which all given conditions return `true` for (AND operation).\n\nSpecify key-value-pairs with the key being the name of the queryable (for STAC APIs) or the metadata property name (for static STAC catalogs). Queryables can be retrieved from the Queryables endpoint of the STAC API and are often but not always aligned with the metadata property names. The value must be a condition (user-defined process) to be evaluated against a STAC API or static catalog. This parameter may not be supported for static STAC catalogs.",
```

some minimal tweaks:

```suggestion
            "description": "Limits the data by metadata properties to include only data in the data cube which all given conditions return `true` for (AND operation).\n\nSpecify key-value-pairs with the key being the name of the queryable (for STAC APIs) or the metadata property name (for static STAC catalogs). Queryables can be retrieved from the Queryables endpoint of the STAC API and are often but not always aligned with the metadata property names. The value must be a condition (user-defined process) to be evaluated against the value of the queryable (for STAC APIs) or metadata property (for static STAc catalogs).\n\nThis parameter may not be supported for static STAC catalogs.",
```

---

And on:

> This parameter may not be supported for static STAC catalogs

I wonder if there should be an explicit note for backend implementers to raise `FeatureUnsupported` if it's not supported, to avoid that it's ignored silently

---

I was confused about the change and first thout on your suggestion, but it was a general issue in the PR so updated the wording correspondigly.